### DISCUSSION TITLE
Map of India that is being shown in leaflet is wrong

### DISCUSSION THREAD

**Discussion by AnzarBaig** [Leaflet/Leaflet#9039] [General]
Map of India that is being shown in leaflet maps is wrong, the correct map is " https://upload.wikimedia.org/wikipedia/commons/thumb/7/74/India-map-en.svg/640px-India-map-en.svg.png "  . Requesting you to change it as soon as possible. If you need any other supporting documents let me know.

**Comment by IvanSanchez** [comment]
Duplicate of https://github.com/Leaflet/Leaflet/issues/5056.
